frappe/core/doctype/communication/test_communication.py

```diff
@@ -259,8 +260,25 @@ def test_parse_emails(self):
 		self.assertEqual(emails[1], "first.lastname@email.com")
 		self.assertEqual(emails[2], "test@user.com")
 
+	def test_signature_in_email_content(self):
+		email_account = create_email_account()
+		signature = email_account.signature
+		comm = frappe.get_doc(
+			{
+				"doctype": "Communication",
+				"communication_medium": "Email",
+				"subject": "Document Link in Email",
+				"sender": "comm_sender@example.com",
+				"content": """<div class="ql-editor read-mode">
+				Hi,
+				How are you?
+				</div>""",
+			}
```

Comment:
Can we add one that tests that signature is not added again in case it exists in the body? To ensure https://github.com/frappe/frappe/pull/16178 (Seems like I forgot to add one back then 🥲)

Reply:
Added via https://github.com/frappe/frappe/pull/17482/commits/ad9092380160e5d69fe07150c506a4df2396b83e